data_storage/export_format.md

```diff
@@ -5,7 +5,7 @@ Export Format
   working directory, with the file name `<profile_name>.tox`
```

Comment:
I'm confused. You are fine with `<profile_name>.tox` being exported into the working directory, but you want `avatars/` be in the export directory?